cypress/features/build/build.feature

```diff
@@ -197,4 +197,9 @@ Feature: Build tests
   @build
   Scenario: Component Draggable basic default page and verify data-element
     When I open basic Test "Draggable" component page in noIframe
-    Then "draggable" element is visible
\ No newline at end of file
+    Then "draggable" element is visible
+
+  @build
+  Scenario: Component Grid full default page
+    When I open "Test Grid" component page full in iframe
```

Comment:
we need to use our convention in naming steps like you've did in accessibility.feature
`Given I open full Test "Grid" component page`